enterprise/tailnet/pgcoord.go

```diff
@@ -27,15 +26,16 @@ import (
 )
 
 const (
-	EventHeartbeats   = "tailnet_coordinator_heartbeat"
-	eventClientUpdate = "tailnet_client_update"
-	eventAgentUpdate  = "tailnet_agent_update"
-	HeartbeatPeriod   = time.Second * 2
-	MissedHeartbeats  = 3
-	numQuerierWorkers = 10
-	numBinderWorkers  = 10
-	dbMaxBackoff      = 10 * time.Second
-	cleanupPeriod     = time.Hour
+	EventHeartbeats         = "tailnet_coordinator_heartbeat"
+	eventClientUpdate       = "tailnet_client_update"
+	eventClientSubscription = "tailnet_client_subscription_update"
```

Comment:
I like the combining of subscription updates with other client updates --- keeps the pubsub logic simpler, but it means that this const is unused and can be deleted.